styles.css

```diff
@@ -0,0 +1,841 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+  display: block;
+}
+
+body {
+  line-height: 1;
+}
+
+ol, ul {
+  list-style: none;
+}
+
+blockquote, q {
+  quotes: none;
+}
+
+blockquote:before, blockquote:after,
+q:before, q:after {
+  content: "";
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.header {
+  background-image: url(./src/images/photos/header.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding: 28px 20px 40px;
+  min-height: 100vh;
+  box-sizing: border-box;
+  grid-template-rows: 1fr 1fr;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, 130px);
+  gap: 20px;
+}
+@media (min-width: 744px) {
+  .header {
+    grid-template-columns: repeat(6, 80px);
+    gap: 24px;
+  }
+}
+@media (min-width: 1260px) {
+  .header {
+    grid-template-columns: repeat(12, 63px);
+    gap: 24px;
+  }
+}
+.header__navigation {
+  grid-column: span 2;
+  grid-row: 1/2;
+  align-self: start;
+}
+@media (min-width: 744px) {
+  .header__navigation {
+    grid-column: span 6;
+  }
+}
+@media (min-width: 1260px) {
+  .header__navigation {
+    grid-column: span 12;
+  }
+}
+.header__title {
+  grid-column: 1/3;
+  grid-row: 2/3;
+  align-self: end;
+}
+@media (min-width: 744px) {
+  .header__title {
+    grid-column: span 6;
+  }
+}
+@media (min-width: 1260px) {
+  .header__title {
+    grid-column: span 12;
+  }
+}
+
+.title {
+  font-family: "Inter";
+  font-style: italic;
+  font-weight: 800;
+  font-size: 32px;
+  line-height: 32px;
+  letter-spacing: -1.5px;
+  color: #292929;
+  position: relative;
+}
+@media (min-width: 744px) {
+  .title {
+    font-size: 40px;
+    line-height: 40px;
+  }
+}
+.title--header::before {
+  content: "";
+  background-image: url(../src/images/icon-sound-waves.svg);
```

Comment:
As with the previous point, consider using a relative path for your background image.